### PR TITLE
Related discussions

### PR DESCRIPTION
**Pull request by sergiorodenas**

### CODE REVIEW

**Comment by sergiorodenas** [comment]
What do you think?

**Review by datlechin — APPROVED** [review]

File: js/src/forum/components/MoreDiscussions.tsx [hunk 1]
```diff
@@ -55,6 +57,9 @@ export default class MoreDiscussions<CustomAttrs> extends Component {
           limit: this.limit,
         },
         sort: this.filterBy,
+        filter: {
+          q: this.isRelated ? app.current.data.discussion.title : null,
```
Comment (datlechin) [inline js/src/forum/components/MoreDiscussions.tsx]: Look like it is `app.current.data.discussion.title()` not `app.current.data.discussion.title`

**Review by datlechin — CHANGES_REQUESTED** [review]

File: js/src/forum/components/MoreDiscussions.tsx (hunk 1 above)
Comment (sergiorodenas) [reply js/src/forum/components/MoreDiscussions.tsx]: Solved

Comment (sergiorodenas) [reply js/src/forum/components/MoreDiscussions.tsx]: Thank you 👍🏼

**Review by datlechin — APPROVED** [review]
Looks good, thank you!

File: js/src/forum/components/MoreDiscussions.tsx (hunk 1 above)
Comment (datlechin) [reply js/src/forum/components/MoreDiscussions.tsx]: Will this also be resolved https://github.com/datlechin/flarum-more-discussions/issues/7?

**Comment by sergiorodenas** [comment]
Should we add a disclaimer that if the option is activated on big sites without https://github.com/clarkwinkelmann/flarum-ext-scout it's going to affect a lot to site performance?

**Comment by datlechin** [comment]
Yeah, I think so. Can you add a notice to the help text of the **Related results** field.

**Pull request merged**